sale_line_returned_qty/tests/test_sale_line_returned_qty.py

```diff
@@ -57,9 +58,24 @@ def test_returned_qty(self):
         # Partial delivery one
         picking = self.order.picking_ids
         picking.action_assign()
-        picking.move_ids.quantity_done = 10.0
+        picking.move_ids.quantity = 10.0
+        picking.move_ids.picked = True
         picking._action_done()
+        self.assertEqual(picking.state, "done")
         self.assertEqual(so_line.qty_returned, 0.0)
         # Make a return for 5 units
-        self._return_picking(picking, 5.0, to_refund=True)
+        stock_return_picking_form = Form(
+            self.env["stock.return.picking"].with_context(
+                active_ids=picking.ids,
+                active_id=picking.ids[0],
+                active_model="stock.picking",
+            )
+        )
+        stock_return_picking = stock_return_picking_form.save()
+        stock_return_picking.product_return_moves.quantity = 5.0
```

Comment:
Moreover, if you use Form(), you should include fields changes using it

Reply:
@rousseldenis Can you please explain this in detail , actually i picked up this process from base.